fvm/evm/stdlib/contract.cdc

```diff
@@ -258,6 +258,10 @@ contract EVM {
         access(all)
         let errorCode: UInt64
 
+        /// error message
+        access(all)
+        let errorMsg: String
```

Comment:
NIT: Should we call it just error since errorCode is more specific this is just general error? If you want to keep message then also maybe `errorMessage` we didn't shorten other words so maybe better for consistency
```suggestion
        let error: String
```